RobotCode/src/main/java/frc/robot/commands/ClimberOverrideCommand.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ClimberSubsystem;
+
+public class ClimberOverrideCommand extends Command
+{
+	private ClimberSubsystem climber;
+
+	public ClimberOverrideCommand(ClimberSubsystem climber)
+	{
+		this.climber = climber;
+		addRequirements(climber);
+		climber.setOverride(true);
```

Comment:
Only actually set the override in Instantiate. We often create commands before we want them to run.